src/pages/trading-specification/components/_elements.tsx

```diff
@@ -229,7 +233,9 @@ export const TableCellGroup = ({ data, market }: TTableCellGroup) => {
     const symbol = data.symbol
     const text = data.instrument
     const dlIcon = data.dl_icon
+    const swfIcon = data.swf_icon
```

Comment:
Can we destructure `data` here? 

```
 const { symbol,instrument: text,dl_icon,swf_icon }
 ```
 
 Also `swfIcon` and `dlIcon` is against to our convention, we should use `snake_case` on this kind of variables.

Reply:
can. made the changes as advised.